index.md

```diff
@@ -86,6 +86,13 @@ Follow these instructions which will guide you through the setup process:
 
 * [Setup your FeatureBase cloud account](/docs/cloud/cloud-home)
 
+## How do I launch FeatureBase with Docker?
+You can quickly start FeatureBase Community using the [Docker Hub](https://hub.docker.com/r/featurebasedb/featurebase) image:
+
+```
+docker run -p 10101:10101 featurebasedb/featurebase
```

Comment:
If I look in Docker Desktop, the image is called "GREAT_ENGLEBART" after it's started.
This could be a joke, but it doesn't help the user determine what the docker container actually is without having to look at the path to make a guess